cookbooks/bcpc-hadoop/templates/default/graphite.query_graphite.py.erb

```diff
@@ -14,7 +14,9 @@ class GraphiteToZabbix(object):
 
   def __init__(self, threads=1):
     self.cn = self.__class__.__name__
-    self.graphite_url = "<%= @graphite_url %>/render?from=-<%= @metric_fetch_period %>min&format=json&target=%s"
+    #self.graphite_url = "<%= @graphite_url %>/render?from=-<%= @default_query_range %>min&format=json&target=%s"
```

Comment:
yes, this commented line should be removed.
I'm working on this file, and it will be cleaned in next PR.
Thanks for the catch.